### ISSUE TITLE
Improve CSS for CLI options listing

### ISSUE THREAD

**Issue by mradamcox** [archesproject/arches-docs#27]
It would be easier to read the CLI docs if the argument tables didn't have a scrollbar, but the text description wrapped instead. Currently styling:
![image](https://user-images.githubusercontent.com/10427268/40985160-1cafebba-68a9-11e8-938a-c9e8e4afc601.png)
An illustration of using dev tools to get the desired styling.
![image](https://user-images.githubusercontent.com/10427268/40984964-abd68fc0-68a8-11e8-99b3-f3892a5f2dff.png)

**Comment by beatboxchad** [comment]
Turning off that white-space directive makes the options malformed (due to the cell size the dash and the letter appear on separate lines on my screen). Ideally we want the options to never wrap, the the descriptions to always. I fiddled around trying to select only one td or another, maybe by means of those columns, but my few minutes of fiddling was unfruitful. At least I got some alliteration out of the deal.

If there's a non-hackey way to get Sphinx to add a custom class to the description td elements, that'll be the way to go about this. I'll look into it.

**Comment by mradamcox** [comment]
Ok sounds good. I would call this enhancement low priority, but nice to have. I certainly don't want to start maintaining a fork of Sphinx or something...

**Comment by beatboxchad** [comment]
Yeah that'd suck. If somebody else has been annoyed by this same thing in the past, and also figured out a pure CSS fix, there might be something to put in requirements.txt other than sphinx_rtd_theme.

Failing that, or a simple directive to sphinx to add custom classes to elements or something else equally elegant, I'll abandon this. I really just picked this up to round out the end of my workday yesterday and my posture toward it will remain the same for maybe one more hour.

**Comment by beatboxchad** [comment]
Skimming the sphinx docs on my phone over late breakfast, it does look very likely that there'll be a simple, maintainable way to get it done. So that's cool.

**Comment by beatboxchad** [comment]
After a few more minutes of research, I think the answer is to not use the fancy option list syntax in ReST, but rather to use a more generic table. There are a bunch of directives in the table syntax you can use to control the class and various other attributes of the generated HTML.

The option list syntax is cool, but it makes a very tight set of assumptions about how options look that Arches doesn't comply with anyway. Using the approach I'm suggesting will make the source slightly more verbose, but more explicit in return so I think it's a good trade-off. Plus we get exactly the look we want instead of fighting the option-list syntax.

PR with added CSS and modified command line reference coming up after some more high-priority work.

**Comment by mradamcox** [comment]
splendid, thanks for looking into that.

**Comment by beatboxchad** [comment]
Yesterday I discovered the table syntax in ReST is not as elegant for this situation as I thought. You suddenly have to manually manage column widths and manually replicate a lot of uniformity. Plus, I was incorrect in assuming that the table syntax would allow me to add a CSS class to one column only.

The problem is that the entire table has one CSS class. There's probably a simple CSS accessor to style each of the columns, but the markup generated by sphinx may not have all the elements required, or I just might suck at CSS. 

At any rate, we want: 

* the first column with the options to NOT wrap, while the second column with the descriptions does. 
* I'd like to get rid of the italic text that appears after the first character of args with only one dash (since Arches breaks the conventions of a single-letter argument having one dash and longer args having two)

It wouldn't hurt my feelings if we tabled this for now and revisited it in the scope of a general cleanup/style standardization..
